java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -131,7 +137,7 @@ public void interruptedExecution(CheckerContext context) {
     public SEIssue(Tree tree, String message, Set<List<JavaFileScannerContext.Location>> flows) {
       this.tree = tree;
       this.message = message;
-      this.flows = flows;
+      this.flows = new HashSet<>(flows);
```

Comment:
It is tested implicitly by using ImmutableSet when constructing the issue. Will be nice to have explicit test, but not worth the effort to test inner private class